apps/src/blockly/addons/cdoUtils.js

```diff
@@ -80,29 +79,27 @@ function prepareSourcesForWorkspaces(source) {
     parsedSource,
     procedureTypesToHide
   );
-  return {mainSource, hiddenDefinitionSource, blockOrderMap};
+  return {mainSource, hiddenDefinitionSource};
 }
 
 /**
  * Convert source to parsed json objects. If source was xml, convert to json before parsing.
  * Also create a block order map if source was xml, which will allow us to correctly place blocks on the workspace.
  * @param {string} source - workspace serialization, either XML or JSON
- * @returns {parsedSource: Object, blockOrderMap: Object}
+ * @returns {parsedSource: Object}
  */
 function parseSource(source) {
   let isXml = stringIsXml(source);
   let parsedSource;
-  let blockOrderMap;
 
   if (isXml) {
     const xml = parseXmlElement(source);
     parsedSource = convertXmlToJson(xml);
-    blockOrderMap = Blockly.Xml.createBlockOrderMap(xml);
   } else {
     parsedSource = JSON.parse(source);
   }
 
-  return {parsedSource, blockOrderMap};
+  return {parsedSource};
```

Comment:
Nit: Is there any reason to continue returning this as an object `{}`?

Reply:
good call, will update